platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -138,6 +152,10 @@ extern const MGLGeoJSONSourceOption MGLGeoJSONSourceOptionSimplificationToleranc
  */
 @property (nonatomic, nullable) NSURL *URL;
 
+/**
+ Returns all objects from the shape that conforms to `MGLFeature`.
```

Comment:
This makes it sound like it does a deep search through the shape collection feature for any shapes, whereas it's only set to the top-level features, and only if a shape collection feature or array of features was passed in on initialization.